data/xml/2019.icon.xml

```diff
@@ -0,0 +1,283 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<collection id="2019.icon">
+  <volume id="1" ingest-date="2021-05-10">
+    <meta>
+      <booktitle>Proceedings of the Sixteenth International Conference on Natural Language Processing</booktitle>
+      <editor><first>Dipti Misra</first><last>Sharma</last></editor>
+      <editor><first>Pushpak</first><last>Bhattacharya</last></editor>
+      <publisher>Association for Computational Linguistics</publisher>
```

Comment:
This should probably be "NLP Association of India" which is what the others have.